### ISSUE TITLE
손등의 일부 뼈에서 성능이 떨어지는 문제

### ISSUE THREAD

**Issue by namgyu-youn** [boostcampaitech7/level2-cv-semanticsegmentation-cv-14-lv3#16]
## Background
1. 기존 FCN model은 2가지 경우에서 성능이 떨어지는 문제가 있었음.
- Case_01 : 손가락 마지막 부위는 학습하기 어려워하는 경우
- Case_02 : "손등에서 Trapezoid, Pisiform 뼈가 다른 뼈와 겹치면서 학습하기 어려워하는 경우

2. Case_01 : U++을 사용해보면서 개선됨 (0.97이상의 높은 성능)
- Model, Backbone, Epochs, loss = U++, efficentnet-b7, 140, BCEWithLogitsLoss 

3. Case_02 : Model, backbone을 바꿔도 여전히 "일부 손등 부위"에서 성능이 떨어지는 모습이 확인됨.
- Trapezoid(0.9103), Pisiform(0.8890)
![{734BAFF9-8171-4846-8390-434CFB1B04BA}](https://github.com/user-attachments/assets/2d4a9352-3db9-4eea-a28d-27a280074684)![{CD49EE40-FC26-49F2-ABC5-7F2D7638873E}](https://github.com/user-attachments/assets/7eb1ad0c-ab63-4752-a58e-123e78ebf8ed)

## To-do
- [x] Class Weight, Augmentation.. 등의 성능 개선 방법론 찾기 (5~6기 기수 분들 자료 활용)

### Checked-list
- [x] #17
- [x] #25

## See also
- Streamlit 기능이 추가되기 전에 실행한 모델이여서 아쉽네요..
- [Wandb log](https://wandb.ai/cv14_gorani/bone_segmentation/runs/1hy5225r/logs)
- [(6th) Semantic Segmentation.pdf](https://github.com/user-attachments/files/17772374/6th.Semantic.Segmentation.pdf)
